kong/clustering/services/sync/rpc.lua

```diff
@@ -269,6 +272,12 @@ local function do_sync()
         return nil, err
       end
 
+      ngx_log(ngx_DEBUG,
+              "[kong.sync.v2] update entity",
+              ", version: ", delta_version,
+              ", type: ", delta_type,
+              ", pk:", cjson_encode(delta.pk))
```

Comment:
Do this lazily!
```suggestion
      if kong.configuration.log_level == "debug"
	      ngx_log(ngx_DEBUG,
	              "[kong.sync.v2] update entity",
	              ", version: ", delta_version,
	              ", type: ", delta_type,
	              ", pk:", cjson_encode(delta.pk))
	     end
```

Reply:
I didn't see this usage in our code base, is it necessary?

Reply:
I think that cjson.encode may be an overhead, should we remove it in the log?

Reply:
> I think that cjson.encode may be an overhead, should we remove it in the log?

I think so. Or if you can know what can present in the pk table, you could do this:

```lua
", pk.id:", delta.pk.id)
```

and it should be safe if id is nil

Reply:
pk is an unknown object, I removed log for it.